### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by 7aretho** [community/community#39268] [GitHub Education]
i have probleme to get the student pack 
when i take photo for my student letter or my student card it say there is a probleme

**Comment by DSDmark** [comment]
Hi @7aretho,

Can you **specific more** about the details what you want. for better understanding 😅. Also you can add some **screenshots** or **error** which you were facing 👍.

**Reply by 7aretho** [reply]
this is the screeshot 
![Screenshot 2022-11-16 201244](https://user-images.githubusercontent.com/103271986/202260177-30167a73-8b34-44be-b661-d58dae7f3600.jpg)

**Reply by 7aretho** [reply]
i want to get student pack

**Comment by DSDmark** [comment]
Hi @7aretho,

first, sorry for late replying. I was kinda trouble with my laptop so,

there is clearly mention as you can see why you are not getting **GitHub developer pack**.

Your school or college, which you're in now. Not partner with **GitHub Campus Program**. I guess.

You can see in which school or college is partner with **GitHub Campus Program**. Visit there:- [**GitHub campus page**](https://education.github.com/schools#close)

First you need to make your profile as mention above image. So, check this out:-[**how to create profile**](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-a-bio-to-your-profile) 

And also try with another document.
